docs/source/changelog.rst

```diff
@@ -11,6 +11,7 @@ Changelog
 * Use version 0.0.25 of `remoto` that fixes an issue where output would be cut
   (https://github.com/alfredodeza/remoto/issues/15).
 * Automatically prefix custom RGW daemon names with 'rgw.'
+* Refuse to install/deploy MDS in RHEL distros as it is not supported.
```

Comment:
I think this line is not quite true anymore.  It's more that we log a warning that it may not work on RHEL if using RHCS, but there is no refusal to do anything.